components/simple-paypal.jsx

```diff
@@ -51,7 +51,12 @@ var simplePaypal = React.createClass({
             </div>
             <div className="row">
               <div className="full">
-                <AmountButtons currencySymbol={this.props.currency.symbol} currency={this.props.currency.code} onChange={this.onAmountChange} amount={amount} presets={this.props.presets} name="amount"/>
+                <AmountButtons name="amount"
+                  currency={this.props.currency}
```

Comment:
We're now passing in the currency as an object with symbol, code, and mine amount instead of passing them in as three props.